itest/rust/src/native_test.rs

```diff
@@ -0,0 +1,141 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use crate::itest;
+use godot::engine::text_server::Direction;
+use godot::engine::{TextServer, TextServerExtension, TextServerExtensionVirtual};
+use godot::native::*;
+use godot::prelude::*;
```

Comment:
Please avoid glob-imports in non-generated code.

Reply:
:+1: